js/bittrex.js

```diff
@@ -383,6 +384,18 @@ module.exports = class bittrex extends Exchange {
         }
         throw new ExchangeError (this.id + ' fetchTrades() returned undefined response');
     }
+    
+    async fetchMyTrades (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        let request = {};
+        let market = undefined;
+        if (symbol) {
+            market = this.market (symbol);
+            request['market'] = market['id'];
+        }
+        let response = await this.accountGetOrderhistory (this.extend (request, params));
```

Comment:
Sorry, but this method is not fetching trades, it is fetching closed orders – and for that we have `fetchOrders()` and `fetchClosedOrders()` below: https://github.com/ccxt/ccxt/pull/1917/files#diff-ca765a07d49756ae1107fcb6fb004461R598

If you look there, you'll notice the implementation is the same as yours, mostly. However, your implementations confuses the trades with closed orders, and the difference is explained here:

- https://github.com/ccxt/ccxt/wiki/Manual#closed-orders

> Do not confuse closed orders with trades aka fills ! An order can be closed (filled) with multiple opposing trades! So, a closed order is not the same as a trade. In general, the order does not have a fee at all, but each particular user trade does have fee, cost and other properties. However, many exchanges propagate those properties to the orders as well.

- https://github.com/ccxt/ccxt/wiki/Manual#trades--transactions--fills--executions

> A trade is a result of order execution. Note, that orders and trades have a one-to-many relationship: an execution of one order may result in several trades. However, when one order matches another opposing order, the pair of two matching orders yields one trade. Thus, when an order matches multiple opposing orders, this yields multiple trades, one trade per each pair of matched orders.